examples/eap/legacy-s2i-source-build/README.md

```diff
@@ -0,0 +1,102 @@
+# Building an EAP 8 application image using EAP 8 S2I legacy source build
+
+In this example we are building and deploying to OpenShift an application that has been migrated to use Jakarta EE 10 
+but has not been updated to use the [EAP Maven Plugin](https://github.com/jbossas/eap-maven-plugin).
+
+# Use-cases
+
+* User wants to keep the EAP 7.4.x S2I workflow. Start from a github source repository, use latest released EAP 8 artifacts and images.
+
+# Helm Chart provisioning configuration
+
+EAP 8 provisioning configuration (feature-packs, layers and channels) is defined in EAP 8 Helm Chart yaml file.
+
+```
+...
+build:
+  s2i:
+    featurePacks:
+      - org.jboss.eap:wildfly-ee-galleon-pack
+      - org.jboss.eap.cloud:eap-cloud-galleon-pack:1.0.0.Alpha8-redhat-00001
+    galleonLayers:
+      - jaxrs-server
+    channels:
+      - org.jboss.eap.channels:eap-8.0-beta
+ ...
+
+High level view of the Helm Chart configuration.
+
+## Galleon feature-packs
+
+* `org.jboss.eap:wildfly-ee-galleon-pack`
+* `org.jboss.eap.cloud:eap-cloud-galleon-pack`
+
+NB, the feature-packs are retrieved from the latest EAP8 channel: `org.jboss.eap.channels:eap-8.0-beta`
+
+## Galleon layers
+
+* `jaxrs-server`
+
+## CLI scripts
+
+JBoss EAP CLI scripts executed at packaging time
+
+* None
+
+## Extra content
+
+Extra content packaged inside the provisioned server
+
+* None
+
+# Openshift build and deployment
+
+Technologies required to build and deploy this example
+
+* TO BE UPDATED WITH EAP/XP helm charts: Helm chart for WildFly `wildfly/wildfly`. Minimal version `2.0.0`.
+
+# Pre-requisites
+
+* You have a `Registry Service Account`. You can [create one](https://access.redhat.com/terms-based-registry/).
+
+* You have downloaded the Openshift secret file allowing to pull images in openshift. For detailed instructions check the URL: https://access.redhat.com/terms-based-registry/#/token/<your user id>/openshift-secret`
+
+* You are logged into an OpenShift cluster and have `oc` command in your path.
+
+* You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
+
+* You have installed the repository for the Helm charts for WildFly
+
+ ```
+helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+```
+
+# Example steps
+
+1. Setup to pull EAP 8 s2i builder and runtime images in Openshift
```

Comment:
this is only required for on-prem installation of OpenShift afaict.

For Red Hat-managed OCP (eg DevSandbox), the secret is automatically attached to the user login...

Reply:
You are correct I just tried to pull EAP 7.4.6 images without particular setup, image was imported. I detailed that in all examples.